### PR TITLE
SPP-10022 example.py

### PR DESCRIPTION
**Pull request by sanjeevz3009**
## Synopsis

Update the existing example.py code to work with the latest thousand pounds method & test if everything is working with the latest 1000s pound method code.

## Checklist

- [x] Documentation created/updated
- [ ] Tests created/updated

## Description

Add a more detailed description of the pr including any necessary background
information.

### CODE REVIEW

**Review by jhmjam — APPROVED** [review]

**Review by jhmjam — DISMISSED** [review]

File: sml_small/editing/thousand_pounds/README.md [hunk 1]
```diff
@@ -91,3 +98,30 @@ Alternatively presented: (mixing input and output data for comparisons)
...
+To install `numpy`:
+
+```python
+pip install numpy
+```
+
+## Pandas Wrapper Usage
+
+- You will have to create a new python file importing in the `pandas_wrapper.py`.
+- Where you will have to have to write functions to read a CSV file and pass in the data as a DataFrame into the *`wrapper`* function from the `pandas_wrapper.py` file.
```
Comment (gibbardsteve) [inline sml_small/editing/thousand_pounds/README.md:125]: have to have to... typo

File: sml_small/editing/thousand_pounds/single_output.csv
Comment (gibbardsteve) [inline sml_small/editing/thousand_pounds/single_output.csv:1]: I think we should gitignore the generated output files.

File: sml_small/editing/thousand_pounds/output1.csv
Comment (gibbardsteve) [inline sml_small/editing/thousand_pounds/output1.csv:1]: gitignore the generated output

**Review by gibbardsteve — COMMENTED** [review]
Some minor comments. Other than that looks good.

File: sml_small/editing/thousand_pounds/single_output.csv
Comment (sanjeevz3009) [reply sml_small/editing/thousand_pounds/single_output.csv:1]: We have the output CSV files for T&C example.py and pandas wrapper output files in the repo. Should I remove and gitignore that too?

Comment (gibbardsteve) [reply sml_small/editing/thousand_pounds/single_output.csv:1]: Yes please, just the output files.

Comment (sanjeevz3009) [reply sml_small/editing/thousand_pounds/single_output.csv:1]: All done =)

File: sml_small/editing/thousand_pounds/output1.csv
Comment (sanjeevz3009) [reply sml_small/editing/thousand_pounds/output1.csv:1]: All done =)

**Review by gibbardsteve — APPROVED** [review]

**Pull request merged**